README.md

```diff
@@ -137,6 +137,7 @@
 - [BetterZip](http://macitbetter.com/) - A very capable and full-featured archive manager.
 - [DaisyDisk](http://www.daisydiskapp.com/) - Analyze disk usage and free up disk space.
 - [Finicky](https://johnste.github.io/finicky/) - App that allows you to set rules that decide which browser is opened for every link. [![Open-Source Software][OSS Icon]](https://github.com/johnste/finicky)
+- [GrandPerspective](http://grandperspectiv.sourceforge.net/) - Graphically show the disk usage within a file system. [![Open-Source Software][OSS Icon]](http://sourceforge.net/p/grandperspectiv/code/HEAD/tree/)
```

Comment:
I just thought because this is the best free tool I found, it might be worth including. But feel free to close the PR if you feel that just the best tool should be on here, regardless of licensing.